src/Analyzer.Core/Rules/BuiltInRules.json

```diff
@@ -1,6 +1,6 @@
 [
   {
-    "name": "AppService_EnableDiagLogs",
+    "id": "TA-000001",
     "description": "Diagnostic logs in App Services should be enabled",
     "recommendation": "Enable diagnostic Logs in App Services",
     "helpUri": "https://github.com/Azure/template-analyzer/blob/main/docs/built-in-bpa-rules.md/#diagnostic-logs-in-app-services-should-be-enabled",
```

Comment:
I think the helpUris need to be updated too since the header changed :(

Reply:
Oh you're right, good catch, luckily I think I can do it in one edit with some Sublime text edition magic